.github/workflows/coverage.yml

```
@@ -2,12 +2,15 @@ name: Post coverage comment
 
 on:
   workflow_call:
-
+    inputs:
+      # this is the workflow run ID of the CI workflow
```

```suggestion
      # this is the workflow run ID of the `CI.yml` workflow
```
same reason as above.